extensions/blocks/story/player/lib/shadow-root.js

```diff
@@ -14,6 +14,15 @@ import { createPortal, useRef, useEffect, useState } from '@wordpress/element';
 export const shadowRootSupported =
 	window && window.Element && window.Element.prototype.hasOwnProperty( 'attachShadow' );
 
+function useHookWithRefCallback() {
```

Comment:
This change is unrelated to this PR though it does improve the code so I'm leaving it here